response_operations_ui/views/collection_exercise.py

```diff
@@ -1010,54 +1025,3 @@ def _delete_seft_collection_instrument(short_name, period):
             success_panel=success_panel,
         )
     )
-
-
-def _upload_seft_collection_instrument(short_name, period):
```

Comment:
Most random prize of the year goes to this one, we have the identical function already, called _upload_collection_instrument. I even used a string comparator on it. The only difference is in the url_for, the params are in a slightly different order, but they are name so that becomes irrelevant